riot/runtime/scheduler/scheduler.ml

```diff
@@ -12,6 +12,7 @@ type t = {
   timers : Timer_wheel.t;
   idle_mutex : Mutex.t;
   idle_condition : Condition.t;
+  mutable currently_stealing : bool;
```

Comment:
Let's make this an `bool Atomic.t` so we guarantee no two schedulers mutate this at the same time.